test/auro-checkbox.test.js

```diff
@@ -1,28 +1,149 @@
-import { fixture, html, expect } from '@open-wc/testing';
-import sinon from 'sinon';
+/* eslint-disable max-statements */
+/* eslint-disable no-console */
+/* eslint-disable one-var */
+/* eslint-disable no-undef */
+/* eslint-disable no-unused-expressions */
+import { fixture, html, expect, elementUpdated } from '@open-wc/testing';
+
 import '../src/auro-checkbox.js';
+import '../src/auro-checkbox-group.js';
+
+describe('auro-checkbox-group', () => {
+  it('has the expected properties', async () => {
+    const expectedError = "Expected error message";
 
-describe('auro-checkbox', () => {
-  it('sets the CSS class on auro-checkbox > div element', async () => {
     const el = await fixture(html`
-      <auro-checkbox cssclass="testClass"></auro-checkbox>
+      <auro-checkbox-group
```

Comment:
Should the label change be tested in here as well?

Reply:
The purpose of a test is to evaluate the code we write. We are not responsible for the functionality of the `<slot>` element, so it's my opinion that we don't need to test that feature.